doc/src/manual/documentation.md

```diff
@@ -897,9 +897,13 @@ following `!!!` syntax:
     This warning admonition has a custom title: `"Beware!"`.
 ```
 
-Admonitions, like most other toplevel elements, can contain other toplevel elements. When no title
-text, specified after the admonition type in double quotes, is included then the title used will
-be the type of the block, i.e. `"Note"` in the case of the `note` admonition.
+If no title text is specified after the admonition type in double quotes,
+then the title used will be the title of the block, i.e. `"Note"` in the case of the `note` admonition.
+
+The full list of natively supported admonition types (which render into distinct colored boxes) can be seen
```

Comment:
I am wondering whether "natively supported" is the best term. Any label is supported, but some get special styling.

Reply:
I'll try top rephrase this.